### PR TITLE
CIF-1364: Show count in productcollection template

### PR DESCRIPTION
**Pull request by laurentiumagureanu**
## Description

Shows product count in PLP and search page

## Related Issue

CIF-1364

## Motivation and Context

Shows product count in PLP and search page

## How Has This Been Tested?

Manually

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/productcollection.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
           data-sly-use.paginationTemplate="paginationbar.html"
           data-sly-use.facetSelectorTemplate="facetselector.html">
 
+<div class="results__count">${productCollection.searchResultsSet.totalResults} items</div>
```
Comment (mhaack) [inline ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/productcollection.html:19]: `items` should be i18n

Also, this line will be rendered if no products are available, but we already have a message for that case.

Comment (cjelger) [reply ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/productcollection.html:19]: Shouldn't we hide that line if no product is available? It's not useful to display "0 items" if we already show the message that there are no products to display.

Comment (herzog31) [reply ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/productcollection.html:19]: It also should be singular `1 item` and plural `2 items`.

Comment (laurentiumagureanu) [reply ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/productcollection.html]: Pushed a change for this

**Pull request merged**